module/PowerShellEditorServices/Commands/Public/CmdletInterface.ps1

```diff
@@ -77,11 +77,13 @@ function Unregister-EditorCommand {
     }
 }
 
-function psedit {
+function Open-EditorFile {
     param([Parameter(Mandatory=$true)]$FilePaths)
 
     dir $FilePaths | where { !$_.PSIsContainer } | % {
```

Comment:
While we're modifying this function, it may be a good time to replace the aliases and use the `-File` parameter (added in PSv3)

```powershell
Get-ChildItem $FilePaths -File | ForEach-Object {
```

Reply:
Good catch. PSSA should have flagged these inside VSCode.  Hmm....

Reply:
It did :) I just missed it. Fixed this!